smac/tae/execute_ta_run.py

```diff
@@ -159,17 +185,21 @@ def start(self, config:Configuration,
                                     "exit. The last incumbent can be found "
                                     "in the trajectory-file.")
 
-        # update SMAC stats
-        self.stats.ta_runs += 1
-        self.stats.ta_time_used += float(runtime)
-
         if self.run_obj == "runtime":
-            if status != StatusType.SUCCESS:
-                cost = cutoff * self.par_factor
-            else:
+            if cutoff is not None and runtime > self.par_factor * cutoff:
+                self.logger.warn("Returned running time is larger "
+                                 "than 2 times the passed cutoff time. "
+                                 "Clamping to {} x cutoff.".format(self.par_factor))
+                runtime = cutoff * self.par_factor
+            if status == StatusType.SUCCESS:
                 cost = runtime
+            else:
+                cost = cutoff * self.par_factor
```

Comment:
Good point. I removed the cutoff=none in the above if-statement and added an error log message if running is used but no cutoff was passed -- this is not intended right now; however, if the target algorithm always returned SUCCESS runs, it would not be a problem.